src/ops/view.rs

```diff
@@ -94,31 +265,36 @@ pub(super) fn pretty_view(
             .map(|v| v.to_string())
             .unwrap_or_else(|| format!("{warn}unknown{warn:#}"))
     )?;
-    if let Some(ref link) = metadata.documentation.clone().or_else(|| {
-        summary.source_id().is_crates_io().then(|| {
-            format!(
-                "https://docs.rs/{name}/{version}",
-                name = package_id.name(),
-                version = package_id.version()
-            )
+    if let Some(link) = metadata
+        .documentation
+        .as_ref()
+        .map(|link| {
+            if is_crate_from_crates_io {
+                Hyperlink::Intrinsic(link)
+            } else {
+                Hyperlink::External(link)
+            }
         })
-    }) {
+        .or_else(|| {
+            is_crate_from_crates_io
+                .then(|| Hyperlink::Documentation(package_id.name(), Some(package_id.version())))
+        })
```

Comment:
A good number of terminals automatically turn URLs into links.  Do we need this?

Reply:
(ditto for other External links)

Reply:
Yeah~

Hmmm... Okay~ I will sunset the transformation about the `already-be-link-format` contents.

Reply:
Hi Ed~

I had sunset the transformations about the already-be-link-format contents.

E.g. `documentation`, `crates.io`, `repository` and the `homepage` properties.